conans/tools.py

```diff
@@ -28,6 +28,7 @@
     rmdir, save as files_save, save_append, sha1sum, sha256sum, to_file_bytes, touch
 from conans.util.log import logger
 from conans.client.tools.version import Version
+from conans.client.build.cppstd_flags import cppstd_flag, cppstd_default  # pylint: disable=unused-import
```

Comment:
IMO, we shouldn't expose `cppstd_default` in this iteration. It can be useful, for sure (there are other issues and PRs asking for it), but I'm not  sure if this function should return `98/11/14/...` or `1998/2003/2011/...`, probably it should return if the implementation is stable or not, (maybe something about gnu extensions?)

Reply:
`cppstd_default` is no longer exposed